bundles/org.openhab.binding.boschshc/src/test/java/org/openhab/binding/boschshc/internal/devices/AbstractPowerSwitchHandlerTest.java

```diff
@@ -103,8 +103,13 @@ public void testUpdateChannelPowerSwitchState() {
 
     @Test
     public void testUpdateChannelPowerMeterServiceState() {
-        JsonElement jsonObject = JsonParser.parseString("{\n" + "  \"@type\": \"powerMeterState\",\n"
-                + "  \"powerConsumption\": \"23\",\n" + "  \"energyConsumption\": 42\n" + "}");
+        JsonElement jsonObject = JsonParser.parseString("""
+                {
+                  "@type": "powerMeterState",
+                  "powerConsumption": "23",
+                  "energyConsumption": 42
+                }\
```

Comment:
what is the purpose of the added `\` here?

Reply:
@david-pace  Putting the closing """ on the next line would add a line break to the string if the backslash is omitted.
I think it is better to read when the string is terminated with """ on a new line.
spotless is fine with both approaches and does not rewrite.